src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+    render() {
+        return (
+            <div className="App">
+                <div className="App-header">
+                    <h2>Car Maitenance Tracker</h2>
```

Comment:
I'm guessing you made this `h2` because it looked nice?  That's one of my problems with using bootstrap/foundation/etc.  It ties the presentation layer to semantic tags.  

Properly it would look something like
```
<h1 class="hdg hdg_medium">Car Maintenance Tracker</h1>
```

Where `.hdg` and `.hdg_medium` would make it look correct.  

The reason this is important is things like screen readers, search engines, and other devices that put specific meaning into tags. 

A screen reader will treat `<h1>` much differently than `<h2>`.  It also means using semantic tags when possible and well supported:

https://github.com/thenerdery/html-css-standards/blob/master/standards/html.md#html5-usage
https://github.com/thenerdery/html-css-standards/blob/master/standards/html.md#semantic-markup